src/ray/object_manager/pull_manager.cc

```diff
@@ -125,17 +134,18 @@ void PullManager::DeactivatePullBundleRequest(
   for (const auto &ref : request_it->second.objects) {
     absl::MutexLock lock(&active_objects_mu_);
     auto obj_id = ObjectRefToId(ref);
-    if (!active_object_pull_requests_[obj_id].erase(request_it->first)) {
-      // If a bundle contains multiple duplicated object ids, the active pull request
-      // could've been already removed. Then do nothing.
+    auto it = active_object_pull_requests_.find(obj_id);
```

Comment:
The use of the `[]` operator caused the errant CHECK failure, since it default-allocated an empty set as the value.